app/views/home_pages/index.html.slim

```diff
@@ -0,0 +1,34 @@
+html[id="Home_html" ]
+body[id="Home_Body"]
+.container
```

Comment:
Diese Zeilen gehören hier nicht rein. Das gehört in die Layouts.
https://guides.rubyonrails.org/layouts_and_rendering.html